mobile/evidence/ios/Features/Tests/ProfileTests/ProfileTests.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  ProfileTests.swift
+//  
+//
+//  Created by Cris Messias on 12/11/23.
+//
+
+import XCTest
+@testable import Profile
+@testable import Models
+
+final class ProfileTests: XCTestCase {
+
+    func testButtonClearToggle() {
+        // Given - Arrange
+        let model = StatusViewModel()
+        
+        // When - Act
+        let initialState =  model.isClearButtonShowing
+        model.toggleButton()
+        let finalState = model.isClearButtonShowing
+        
+        // Then - Assert
+        XCTAssertNotEqual(initialState, finalState, "Not equal")
```

Comment:
Instead of asserting not equality, it would be more clear if you assert the actual values you expect. How about this?
```suggestion
        XCTAssertEqual(initialIsClearButtonShowing, true)
        XCTAssertEqual(finalIsClearButtonShowing, false)
```